boards/x86/rpl_crb/rpl_crb_defconfig

```diff
@@ -13,3 +13,4 @@ CONFIG_X2APIC=y
 CONFIG_SMP=y
 CONFIG_BUILD_OUTPUT_EFI=y
 CONFIG_BUILD_NO_GAP_FILL=y
+CONFIG_UART_NS16550_AUTO_INIT_LEVEL=y
```

Comment:
What happens to other boards, like up_squared and ehl_crb, when your ACPI PR gets merged? Will they be broken?